src/system/utility.ts

```diff
@@ -39,4 +62,7 @@ export type UtilityProps = {
 
   /** Makes the component invisible to the eye, but still readable by screen readers */
   visuallyHidden?: boolean;
+
+  /** Adds an opacity to the existing `backgroundColor` */
+  backgroundOpacity?: string | number;
```

Comment:
As an alternative, you could make this an enumeration `[0,5,10...]` and/or `[0.25,0.5 ... ]` more restrictive but I don't see much value on having a custom opacity i.e. `0.63`

Reply:
Neither do I. I also don't want to add input validation at this level :/